sympy/physics/mechanics/tests/test_lagrange.py

```diff
@@ -64,7 +64,7 @@ def test_simp_pen():
     # inertial frame.
     q, u = dynamicsymbols('q u')
     qd, ud = dynamicsymbols('q u ', 1)
-    l, m, g = symbols('l m g')
+    l, m, g = symbols('l m g', real=True)
```

Comment:
@isuruf Though I'm not sure why this happens, but removing `real=True` argument fixes the failure for `test_lagrange.py`. This is probably because at the time of using `sympify` over these objects, the resulting test expression isn't simplified. As far as I know, the `Symbol` class in `SymEngine.py` does take extra arguments but nothing is done of them.